administrator/language/de-DE/com_media.sys.ini

```diff
@@ -5,6 +5,6 @@
 ; Note : All ini files need to be saved as UTF-8
 
 COM_MEDIA="Medien"
-COM_MEDIA_MEDIA_VIEW_DEFAULT_DESC="Bilder und andere Mediendateien auf der Website verwalten."
-COM_MEDIA_MEDIA_VIEW_DEFAULT_TITLE="Medien"
+COM_MEDIA_MEDIA_VIEW_DEFAULT_DESC="Bilder und andere Mediendateien auf der Joomla! Website verwalten."
+COM_MEDIA_MEDIA_VIEW_DEFAULT_TITLE="Media Manager"
```

Comment:
Wollen wir den bewusst in Media Manager umbenennen? Oder soll es Medien bleiben

Reply:
Ich habe auf den JDOC-Translate Seiten bisher den Begriff "Medien-Manager" verwendet.

Reply:
Wir hatten das fürs Frontend halt so, hab das nur übernommen und gleich gezogen.

Finde Medien-Manager auch besser.

Dann mache ich da auch einen PR noch fürs Frontend.